tests/spanner_dbapi/test_connect.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+"""connect() module function unit tests."""
+
+import unittest
+from unittest import mock
+
+
+def _make_credentials():
+    import google.auth.credentials
+
+    class _CredentialsWithScopes(
+        google.auth.credentials.Credentials, google.auth.credentials.Scoped
+    ):
+        pass
+
+    return mock.Mock(spec=_CredentialsWithScopes)
+
+
+class Testconnect(unittest.TestCase):
+    def _callFUT(self, *args, **kw):
+        from spanner_dbapi import connect
+
+        return connect(*args, **kw)
+
+    def test_connect(self):
+        from google.api_core.gapic_v1.client_info import ClientInfo
+        from spanner_dbapi.connection import Connection
+
+        PROJECT = "test-project"
+        USER_AGENT = "user-agent"
+        CREDENTIALS = _make_credentials()
+        CLIENT_INFO = ClientInfo(user_agent=USER_AGENT)
```

Comment:
Nit: why the caps if these are just regular non-global vars?

Reply:
It's what they do in other clients ([example](https://github.com/googleapis/python-spanner/blob/45a15382bc1e62dedc944f6484c15ba929338670/tests/unit/test_database.py#L213-L214)). I can't say why it's done like this, but I'm okay with both variants.